tornado/test/options_test.py

```diff
@@ -24,6 +24,13 @@
         mock = None
 
 
+class email(str):
```

Comment:
If we use a factory function wich return a sting, we must define string typed option, but we 
can set any string using (options.email = "this isn't an email") I therefore prefer use distinct class. Using subclass of str is a possible case but I do no have raison to use it instead of plain class.